orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt

```diff
@@ -316,6 +316,9 @@ class RunTaskHandler(
    * `tasks.aws.backOffPeriod = 80000`
    * `tasks.aws.someAccount.backoffPeriod = 60000`
    * `tasks.aws.backoffPeriod` will be used (given the criteria matches and unless the default dynamicBackOffPeriod is greater).
+   *
+   * This function also considers the `backoffPeriodMs` property in a stage
+   * configuration.  If it's larger than the above, it's used.
```

Comment:
Im not following this comment. Where is the logic to determine what is used by magnitude?

Reply:
Nvm. I see it in the code below that needed expanding in gh